packages/client/modules/outcomeCard/components/OutcomeCardAssignMenu/TaskFooterTeamAssigneeAddGitHubDialog.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import {PALETTE} from '~/styles/paletteV3'
+import {ICON_SIZE} from '~/styles/typographyV2'
+import styled from '@emotion/styled'
+import Icon from '~/components/Icon'
+import DialogContainer from '~/components/DialogContainer'
+import DialogTitle from '~/components/DialogTitle'
+import SecondaryButton from '~/components/SecondaryButton'
+import useAtmosphere from '~/hooks/useAtmosphere'
+import useMutationProps from '~/hooks/useMutationProps'
+import PrimaryButton from '~/components/PrimaryButton'
+import DialogContent from '~/components/DialogContent'
+import GitHubClientManager from '~/utils/GitHubClientManager'
+
+interface Props {
+  onClose: () => void
+  onIntegrationAdded: () => void
+  teamId: string
+  teamName: string
+}
+
+const StyledDialogContainer = styled(DialogContainer)({
+  width: 480
+})
+
+const ButtonGroup = styled('div')({
+  marginTop: '24px',
+  display: 'flex',
+  justifyContent: 'flex-end'
+})
+
+const StyledTip = styled('p')({
+  fontSize: 13,
```

Comment:
I find the text here a little bit hard to read. Perhaps fontSize 14 with lineHeight 20?

Reply:
Yes, looks much better. It came from `AddMissingJiraFieldModal` so we might want to increase size there as well